app/src/main/java/com/amaze/filemanager/filesystem/compressed/extractcontents/Extractor.java

```diff
@@ -97,6 +97,9 @@ public void extractEverything() throws IOException {
     protected String fixEntryName(String entryName){
         if(entryName.indexOf('\\') > 0) {
             return entryName.replaceAll("\\\\", CompressedHelper.SEPARATOR);
+        } if(entryName.indexOf(CompressedHelper.SEPARATOR_CHAR) == 0) {
+            //if entryName starts with "/" (e.g. "/test.txt"), strip the prefixing "/"s
+            return entryName.substring(entryName.lastIndexOf(CompressedHelper.SEPARATOR_CHAR)+1);
```

Comment:
Suppose we have a file named "/file/" and we uncompress it from a zip, we would end up with "" for a name, I think.